install/sql_files/api_keys.sql

```diff
@@ -0,0 +1,12 @@
+-- API keys table
+create table api_keys (
+        id          int  primary key default nextval('id_seq'),
```

Comment:
Any reason to use the shared id_seq here? How about we use `id SERIAL PRIMARY KEY`? It would be less coupled to the other tables without any downside that I can think of.